compliance_suite/functions/requestor.py

```diff
@@ -0,0 +1,71 @@
+import json
+import requests
+from compliance_suite.constants.constants import REQUEST_HEADERS
+import polling2
+from compliance_suite.exceptions.ComplianceException import ComplianceException
+
+check_cancel = None
```

Comment:
I'd rather create a class for the client and pass this in the constructor. Using `global` always makes me a bit uneasy.